iGossip/app/base/settings.py

```diff
@@ -88,7 +88,7 @@
     'default': {
         'ENGINE': 'django.db.backends.mysql',
         'OPTIONS': {
-            'read_default_file': 'base/mysql.conf',
+            'read_default_file': '/Users/gauravsharma/CS411/mysql.conf',
```

Comment:
Moe and I are both using base/ as the dir for config. May be using the same dir for conf will make things more clear since you'll not need to merge this file every time